docs/.vuepress/theme/global-components/InstallLinkAlt.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+	<router-link :to="link" class="install-link install-link--alt">
+    <span class="install-link__container">
+      <span class="icon__wrapper">
+        <slot name="icon" />
+      </span>
+      <span class="title__wrapper">
+        <p class="title"><slot name="title" /></p>
+			<!-- <p class="description"><slot name="description" /></p> -->
+      </span>
+    </span>
+
+	</router-link>
+</template>
+
+<script>
+	export default {
+		name: 'InstallLinkAlt',
+		props: {
+			link: {
+				type: String,
+				required: true
+			}
+		}
+	}
+</script>
+
+<style lang="scss" scoped>
```

Comment:
FYI, this CSS and the overall design of these links will be reviewed & improved in collaboration with designers & front end devs :-)